www/unstable/userguide/weekyears.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: userguide
+title: Week years
+category: advanced
+weight: 3037
+---
+
+Introducing week-years
+===
+
+Most of the time, dates are represented as a year, month and day in
+a particular calendar system - occasionally with an era as well. An
+alternative approach is to represent a date as a week-year, week,
```

Comment:
perhaps "as a year, week, and day-of-week" here, then a following sentence to introduce the idea that we have to use a slightly different year (called a week-year) for some cases?